fractions/microprofile/microprofile-jwt/src/main/java/org/wildfly/swarm/microprofile/jwtauth/deployment/auth/config/JWTAuthContextInfoProvider.java

```diff
@@ -48,7 +48,6 @@
 
     @PostConstruct
     void init() {
-        System.out.printf("JWTAuthContextInfoProvider.init\n");
```

Comment:
Maybe we should remove the whole method or use a regular logger instead of `System.out`?

Reply:
Thought about removing method completely, but left for now